alephclient/cli.py

```diff
@@ -237,6 +237,19 @@ def read_json_stream(stream):
         raise click.Abort()
 
 
+@cli.command("delete-entity")
+@click.argument("entity_id", required=True)
+@click.pass_context
+def delete_entity(ctx, entity_id):
```

Comment:
Perhaps here as well (if the above works)?

```suggestion
def delete_entity(ctx, entity_id: int):
```

Reply:
Same as before - this isn't an integer, but a string that is a hash. How would you recommend I annotate it?

Reply:
perhaps as `str`, that's likely the best we can do here.